visual-testing-app/src/components/custom-form-modal-page/custom-form-modal-page.visualroute.tsx

```diff
@@ -17,6 +17,12 @@ type ContainerProps = {
 type FormValues = {
   email: string;
 };
+type CustomFormikErrorsField = {
+  [errorKey: string]: boolean; // <-- our ui-kit components use a boolean flag to indicate the error
+};
+type CustomFormikErrors<Values = FormikValues> = {
+  [K in keyof Values]?: CustomFormikErrorsField;
+};
```

Comment:
This is more of a general "issue" with the `errors` object different shape of Formik and of our Field components.

This is how it's done now:

```ts
errors={(formikProps.errors as CustomFormikErrors<FormValues>).email}
```

We might want to expose something from uikit field components though. I think a static method might also be enough instead of letting users casting types.

```ts
error={toFieldErrors<FormValues>(formikProps.errors).email}
```

Reply:
See PR that implements this: https://github.com/commercetools/ui-kit/pull/2141